pages/dkp/kommander/1.3/include/kba-requirements.tmpl

```diff
@@ -0,0 +1,406 @@
+## Kubernetes Base Addons Requirements
+
+Kubernetes Base Addons (KBA) require more resources than solely deploying clusters. Some addons are prerequisites for others to deploy. Your cluster must have sufficient resources when deploying a Konvoy cluster, or you must adjust the addons deployed before running `konvoy up`.
+
+The following table describes all KBAs supplied, by default, with Konvoy, minimum resource suggestions and persistent storage requirements, and their default status when starting Konvoy.
+
+<table>
```

Comment:
I strong urge using Github flavoured markdown here to create this table as opposed to using raw-html.